src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.test.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import faker from 'faker';
+import times from 'lodash/times';
+
+import { createEnsObject } from 'tests/fixtures/ens-object';
+import {
+  TrackPanelBookmarks,
+  PreviouslyViewedLinks,
+  ExampleLinks
+} from './TrackPanelBookmarks';
+
+import ImageButton from 'src/shared/components/image-button/ImageButton';
+import { PreviouslyViewedObject } from '../../trackPanelState';
+
+jest.mock('react-router-dom', () => ({
+  Link: (props: any) => (
+    <div {...props} className={'link'}>
+      {props.children}
+    </div>
+  )
+}));
+
+const createPreviouslyViewedLink = (): PreviouslyViewedObject => ({
+  genome_id: faker.random.word(),
+  object_id: faker.random.word(),
+  object_type: faker.random.word(),
+  label: faker.random.word()
+});
+
+const closeTrackPanelModalMock = jest.fn();
+const updateTrackStatesAndSaveMock = jest.fn();
+const fetchExampleEnsObjectsMock = jest.fn();
+const changeDrawerViewAndOpenMock = jest.fn();
+
+describe('<TrackPanelBookmarks />', () => {
+  const numberOfExampleObjects = faker.random.number({ min: 5, max: 10 });
+  const numberOfPreviouslyViewedObjects = faker.random.number({
+    min: 5,
+    max: 20
+  });
+
+  const props = {
+    activeGenomeId: faker.random.word(),
+    exampleEnsObjects: times(numberOfExampleObjects, () => createEnsObject()),
+    previouslyViewedObjects: times(numberOfPreviouslyViewedObjects, () =>
+      createPreviouslyViewedLink()
+    ),
+    fetchExampleEnsObjects: fetchExampleEnsObjectsMock,
+    updateTrackStatesAndSave: updateTrackStatesAndSaveMock,
+    closeTrackPanelModal: closeTrackPanelModalMock,
+    changeDrawerViewAndOpen: changeDrawerViewAndOpenMock
```

Comment:
These can use the shortened format as both the property names and value match.

Reply:
That's done